test/stripe_test.rb

```diff
@@ -23,28 +23,112 @@ class StripeTest < Test::Unit::TestCase
     end
   end
 
-  should "allow ca_bundle_path to be configured" do
-    begin
-      old = Stripe.ca_bundle_path
-      Stripe.ca_bundle_path = "path/to/ca/bundle"
-      assert_equal "path/to/ca/bundle", Stripe.ca_bundle_path
-    ensure
-      Stripe.ca_bundle_path = old
+  context "forwardable configurations" do
```

Comment:
These specs are slightly redundant, but they're cheap and lock in the expected API on `Stripe` for configuration.